### DISCUSSION TITLE
Fail to login at web browser

### DISCUSSION THREAD

**Discussion by zhangh43** [concourse/concourse#8227] [Help & Support]
I started `concourse web --external-url=https://34.92.171.220  ....` successfully
And I can use `fly` to login with username/password.
But when I open the web browser and type `login` it hung and then report `This site can’t be reached34.92.171.220 took too long to respond`
 Here is the link after I click login.
```
https://34.92.171.220/sky/issuer/auth?access_type=offline&client_id=concourse-web&redirect_uri=https%3A%2F%2F34.92.171.220%2Fsky%2Fcallback&response_type=code&scope=openid+profile+email+federated%3Aid+groups&state=eyJyZWRpcmVjdF91cmkiOiIvIiwiZW50cm9weSI6IjI0YzZmZDQxOTFjMmZkNTYxMTAwZjNiZjdhM2FjMzlhMjc5NzE0NDE2N2M4ZDdmNmVlZTIyYzg5NDM0MGVhODUifQ%3D%3D
```

How do I setup the user/password authentication?

**Comment by ln-rinterface** [comment]
I actually had the same question sorta, I installed the newest version 7.7.1 and all of a sudden it went from login w/ username now its email address. Locked out at the moment.

**Reply by taylorsilva** [reply]
The email address thing is a cosmetic change. You can type in a username and it'll work. Not sure where this change came from, might be from upstream dex. @xtremerui would probably know.

**Reply by xtremerui** [reply]
do you mean you input username prior of 7.7.1 upgrade now you need to input email address? What version of Concourse did you upgrade from?

**Comment by taylorsilva** [comment]
Did you take a look at the web logs? There may be some errors there. There is a separate auth server that maybe didn't come up correctly?
